src/util/Util.js

```diff
@@ -193,6 +194,21 @@ define(['okta', './Logger', './Enums'], function (Okta, Logger, Enums) {
     return explain;
   };
 
+  Util.getRouterFromResponse = function (response) {
+    //response.version = '1.0.0';
+    var Router = response.version && response.version === '1.0.0' ? require('v2/WidgetRouter'): require('LoginRouter');
+    return Router;
+  };
+
+  Util.introspectToken = function (authClient, widgetOptions) {
+    return Model.prototype.evaluate(function () {
+      return authClient.tx.evaluate({
```

Comment:
We could call `authClient.tx.evaluate` directly. I chose `evaluate` since its a method that will do an `introspect` or `evaluate` the pipeline for the stateToken based on a flag.